presto-parquet/src/main/java/com/facebook/presto/parquet/batchreader/BooleanFlatBatchReader.java

```diff
@@ -255,3 +255,4 @@ private void seek()
         }
     }
 }
+
```

Comment:
adding these blank lines should be a separate PR, if it's needed at all. (If checkstyle doesn't complain it probably isn't)

Reply:
They are added automatically by the templating tool which generates the files, but weren't caught before because they existed in `generated-sources`. I tried a large combination of [whitespace-control directives from freemarker](https://freemarker.apache.org/docs/dgui_misc_whitespace.html) but was unsuccessful in removing them. I can split these changes into a separate commit on this PR. The commit will be preserved when this PR merges

Reply:
I'd go even further and make this a separate PR that goes in first since there are no other changes in this file.

Reply:
What's the benefit of making a separate PR? Commits in a PR are preserved after the merge